assets/js/calypsoify.js

```diff
@@ -0,0 +1,43 @@
+( function( $ ) {
+    'use strict';
+    
+    /**
+     * Move admin notices
+     */
+    $( document ).ready( function() {
+        $( 'body' ).append( '<div class="wc-calypso-bridge-notices"></div>' );
+        $( 'div.updated, div.error, div.notice' ).not( '.inline, .below-h2' ).appendTo( $( '.wc-calypso-bridge-notices' ) );
+    } );
+
+    /**
+     * Append icons to notices
+     */
+    $( 'div.notice, div.error, div.updated, div.warning' ).each( function() {
+        var icon = '<svg class="gridicon gridicons-info" height="24" width="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g><path d="M12 2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"/></g></svg>';
+        if ( $( this ).hasClass( 'notice-success') ) {
+            icon = '<svg class="gridicon gridicons-checkmark" height="24" width="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g><path d="M9 19.414l-6.707-6.707 1.414-1.414L9 16.586 20.293 5.293l1.414 1.414"/></g></svg>';
+        } else if ( $( this ).hasClass( 'error' ) || $( this ).hasClass( 'notice-warning' ) ) {
+            icon = '<svg class="gridicon gridicons-notice" height="24" width="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g><path d="M12 2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2zm1 15h-2v-2h2v2zm0-4h-2l-.5-6h3l-.5 6z"/></g></svg>';
+        }
+        $( this ).prepend( '<span class="wc-calypso-bridge-notice-icon-wrapper">' + icon + '</span>' );
+    } );
+
+    /**
+     * Replace dismissal buttons in notices
+     */
+    $( document ).ready( function() {
+        $( '.notice-dismiss' ).html( '<svg class="gridicon gridicons-cross" height="24" width="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g><path d="M18.36 19.78L12 13.41l-6.36 6.37-1.42-1.42L10.59 12 4.22 5.64l1.42-1.42L12 10.59l6.36-6.36 1.41 1.41L13.41 12l6.36 6.36z"/></g></svg>' );
```

Comment:
I don't think we should add dismissal buttons unless we can persist these across page views. I think it would be fine to just remove this for now.

Reply:
There are some notices that do have persistable dismiss buttons. The placement of the 'X' is off for those though

Here's a screenshot:

<img width="1313" alt="screen shot 2018-11-01 at 11 10 05 am" src="https://user-images.githubusercontent.com/689165/47860367-bed9ad80-ddc6-11e8-86b9-a7a706ef3ecd.png">

Reply:
Nice find!  I added some styling to override the WooCommerce dismiss styles. 

This JS doesn't make any assumptions or add dismiss buttons if they don't exist; it's only responsible for replacing the dismiss button with the SVG used in Calypso so I think it's safe to use- do you agree?

Reply:
> it's only responsible for replacing the dismiss button with the SVG used in Calypso so I think it's safe to use- do you agree?

Sorry my bad. I think I just confused myself during testing because some didn't persist, but you're right that this just replaces them. Non-persisting notices are the plugins fault, so this line is fine with me :).